x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -181,11 +181,6 @@ export const getBreadcrumbsForRoute = (
   }
 
   if (isAdminRoutes(spyState) && object.navTabs) {
-    const tempNav: SearchNavTab = { urlKey: 'administration', isDetailPage: false };
```

Comment:
you really wanted to delete that

Reply:
yes, sorry I forgot to explain this change here. It is an unrelated cleaning from this @stephmilovic comment: https://github.com/elastic/kibana/pull/103927#discussion_r669899431